board/spark-config.cmake

```diff
@@ -0,0 +1 @@
+
```

Comment:
`spark` or `sparc`?  The old Sun boxes under my desk are getting excited :smile:

Reply:
:rofl: That's what happens when working with robots for a long time, `Sparkfun`
I actually forget an important file